### PR TITLE
choose cluster if not provided

### PR DESCRIPTION
**Pull request by wreed4**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -35,9 +36,7 @@ require (
 	github.com/gravitational/configure v0.0.0-20160909185025-1db4b84fe9db
 	github.com/gravitational/form v0.0.0-20151109031454-c4048f792f70
 	github.com/gravitational/kingpin v2.1.11-0.20190130013101-742f2714c145+incompatible
-	github.com/gravitational/license v0.0.0-20180912170534-4f189e3bd6e3
 	github.com/gravitational/oxy v0.0.0-20200916204440-3eb06d921a1d
-	github.com/gravitational/reporting v0.0.0-20180907002058-ac7b85c75c4c
```
Comment (wreed4) [inline go.mod:40]: these changes don't seem right, but they happened when I ran "make update-vendor" to include `survey`.

Comment (webvictim) [reply go.mod:40]: I think this is because you don't have the Enterprise source available to you.

**Comment by webvictim** [comment]
@wreed4 Thanks for the PR! I like the idea of making `tsh` a bit more interactive in general - being able to select a cluster from a list at the command line is definitely more friendly than the current way. Could you extend this PR to cover Teleport SSH clusters as well for consistency?

Once approved, we can take it over, rerun `go mod vendor` against the Enterprise source and merge it.

**Comment by wreed4** [comment]
@webvictim Is that when you do `tsh login` and it's supposed to list the output of `tsh clusters`? Or `tsh ssh` and it's supposed to be the output of `tsh ls`?  The latter seems like you must not be able to use a raw choice as you have the `user@` portion to contend with.  Sorry, I'm not as familiar with the non-kube use-cases, but if you point me at the pair of commands, I'd be happy to add this.

**Comment by wreed4** [comment]
`lib/client/api.go:2049`  is a similar error message to the kube use-case. Is this where you'd want this?

**Comment by webvictim** [comment]
Yes, it's similar. I wouldn't worry about it for now - we'll let other engineers review this PR as-is and see whether they have additional feedback.

**Comment by russjones** [comment]
@wreed4 Thank you for your contribution, unfortunately this is not a direction we're going to be going in at the moment, if anything changes we'll do our best to reference your work.

**Pull request closed without merging**